unitex/src/fr/umlv/unitex/frames/GraphPathDialog.java

```diff
@@ -249,6 +323,20 @@ private JPanel constructGraphNamePanel() {
 		public void toDo(boolean success) {
 			textArea.load(name);
 			textArea.getModel().addListDataListener(listListener);
+			
+			try {
+				// issue #61 - recursive path option invokes UnitexToolLogger which hard codes the name of the output file to GraphNameautolst.txt 
+				// once that process has completed and loaded the file rename it using the user input if that differs from the default
+				if (!name.getAbsolutePath().equals(outputFileName.getText())) {
+					File dest = new File(outputFileName.getText());
+					Files.move(name.toPath(), dest.toPath(), StandardCopyOption.REPLACE_EXISTING);
+				}
+			}  catch (final IOException e) {
+				e.printStackTrace();
+				JOptionPane.showMessageDialog(null,
+						"Could not save file with name [" + outputFileName.getText() + "] a file has been created [" + name.getAbsolutePath() + "]", "Error",
```

Comment:
Given `/wrong/name.txt`, What about of  `Could not save path list to /wrong/name.txt`